package.json

```diff
@@ -28,13 +27,15 @@
     "precommit": "lint-staged",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "start": "react-scripts start",
+    "start": "npm run check-version && react-scripts start",
     "build": "react-scripts build && cpy build/index.html build --rename 404.html",
     "lint": "eslint .",
+    "preinstall": "npm run check-version",
     "postinstall": "npm install --prefix server",
     "setup": "node ./bin/auth0setup/promptUser.js",
     "gc": "cd ./server && dotenv -e ../.env.local graphcool",
-    "deployProdBackend": "cd ./server && graphcool deploy"
+    "deployProdBackend": "cd ./server && graphcool deploy",
+    "check-version": "node ./src/utils/checkVersion.js"
```

Comment:
I would probably put this in the `bin` directory as a script that is associated with setup, whereas the way this repo is set up the `src` directory is where the client code lives.

Reply:
Good point, not sure how I missed /bin before. Isn't the usual convention to put CLI scripts in /.bin?